torchvision/csrc/DeformConv.h

```diff
@@ -105,14 +121,18 @@ class DeformConv2dFunction
       int64_t dilation_w,
       int64_t groups,
       int64_t offset_groups) {
-    auto output = DeformConv2d_forward(
+    at::AutoNonVariableTypeMode g; // TODO_vv: check if necessary
```

Comment:
This is present in RoIAlign, so I think I'd keep it.

Even though it appears that `g` this is not being used, it's a [RAII class](https://en.cppreference.com/w/cpp/language/raii)